src/main/java/com/machowina/controller/DriverController.java

```diff
@@ -0,0 +1,53 @@
+package com.machowina.controller;
+
+import java.math.BigDecimal;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.machowina.service.TicketService;
+
+@RestController
+@RequestMapping("/drivers")
+public class DriverController {
+	
+	
+	private final TicketService ticketService;
+	
+	@Autowired
+	public DriverController(TicketService ticketService) {
+		super();
+		this.ticketService = ticketService;
+	}
+	
+
+	@PostMapping("/cars/{carId}/zones/{zoneId}/newTicket")
+	@ResponseStatus(HttpStatus.CREATED)
+	public Long generateTicket(@PathVariable Long carId, @PathVariable Long zoneId) {
```

Comment:
czemu duży Long? to pytanie też do innych endpointów

Reply:
Trzymam id w dużym Longu, więc przekazywałam do endpoitu też w takiej postaci. Ale wydaje mi się że równie dobrze może tu być użyty mały long, bo nie korzystamy tu w żaden sposób z jego obiektowości